lib/tether/config.go

```diff
@@ -158,6 +158,8 @@ type NetworkEndpoint struct {
 	IP *net.IPNet `vic:"0.1" scope:"read-only" key:"ip"`
 
 	// Actual IP address assigned
+	// TODO: should this skip decode? - if it's used to seed DHCP requests for semi-stable addressing then
```

Comment:
Should we file a ticket to track this TODO?

Reply:
Will remove the TODO - with the benefit of time I can recall that the non-persistent keys will never be written from the API side while the VM is powered on. This would only become necessary if we were to change that AND had a reason to update this field from the API side (I cannot think of one).